cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/teleport.rs

```diff
@@ -80,3 +80,102 @@ fn teleport_via_transfer_assets_from_and_to_relay() {
 		transfer_assets
 	);
 }
+
+/// Limited Teleport of native asset from System Parachain to Asset Hub
+/// shouldn't work when there is not enough balance in Asset Hub's `CheckAccount`
+#[test]
+fn limited_teleport_native_assets_from_relay_to_asset_hub_checking_acc_fails() {
```

Comment:
not needed, scenario already covered by AH teleport tests

(test name is also wrong)